pkg/controller/servicebindingrequest/sbrcontroller_test.go

```diff
@@ -121,55 +173,53 @@ func TestSBRControllerBuildGVKPredicate(t *testing.T) {
 			wanted bool
 			a      runtime.Object
 			b      runtime.Object
-			metaA  metav1.Object
-			metaB  metav1.Object
 		}{
 			{
 				name:   "non supported update",
 				wanted: false,
 				a:      deploymentA,
 				b:      deploymentA,
-				metaA:  deploymentA.GetObjectMeta(),
-				metaB:  deploymentA.GetObjectMeta(),
 			},
 			{
 				name:   "supported update no changes",
 				wanted: false,
 				a:      configMapA,
 				b:      configMapA,
-				metaA:  configMapA.GetObjectMeta(),
-				metaB:  configMapA.GetObjectMeta(),
 			},
 			{
 				name:   "supported update no changes",
 				wanted: false,
 				a:      secretA,
 				b:      secretA,
-				metaA:  secretA.GetObjectMeta(),
-				metaB:  secretA.GetObjectMeta(),
 			},
 			{
-				name:   "supported update generation changed",
+				name:   "supported update configMap data changed",
 				wanted: true,
 				a:      configMapA,
 				b:      configMapB,
-				metaA:  configMapA.GetObjectMeta(),
-				metaB:  configMapB.GetObjectMeta(),
 			},
 			{
-				name:   "supported update generation changed",
+				name:   "supported update secret data changed",
 				wanted: true,
 				a:      secretA,
 				b:      secretB,
-				metaA:  secretA.GetObjectMeta(),
-				metaB:  secretB.GetObjectMeta(),
+			},
+			{
+				name:   "supported update deployment status changed",
```

Comment:
can you please more specific about what is exactly supported here, i.e. what is the test context.

Reply:
Done in a previous commit